resources/definitions/renkforce_rf100.def.json

```diff
@@ -0,0 +1,231 @@
+{
+    "id": "RF100",
+    "version": 2,
+    "name": "renkforce RF100",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "author": "Simon Peter (based on RF100.ini by Conrad Electronic SE)",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "manufacturer": "renkforce",
+        "platform_offset": [0, 0, 0],
+        "visible": true
+    },
+
+    "overrides": {
+        "adhesion_type": {
+            "default_value": "skirt"
+        },
+        "bottom_thickness": {
+            "default_value": 0.5
+        },
+        "brim_line_count": {
+            "default_value": 20.0
+        },
+        "cool_fan_enabled": {
+            "default_value": true
+        },
+        "cool_fan_full_at_height": {
+            "default_value": 0.5
+        },
+        "cool_fan_speed_max": {
+            "default_value": 100.0
+        },
+        "cool_fan_speed_min": {
+            "default_value": 100.0
+        },
+        "cool_lift_head": {
+            "default_value": true
+        },
+        "cool_min_layer_time": {
+            "default_value": 5.0
+        },
+        "cool_min_speed": {
+            "default_value": 10.0
+        },
+        "infill_before_walls": {
+            "default_value": 1.0
+        },
+        "infill_overlap": {
+            "default_value": 15.0
+        },
+        "infill_sparse_density": {
+            "default_value": 15.0
```

Comment:
Are you sure that the infill density of this printer should be different from the default infill density of other printers?